### PR TITLE
LPS-157363

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related ticket: https://issues.liferay.com/browse/LPS-157363

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java [hunk 1]
```diff
@@ -923,11 +926,14 @@ protected void updateDDMContent(
 	}
 
 	protected void updateFormInstanceRecordVersion(
-		User user, DDMFormInstanceRecordVersion ddmFormInstanceRecordVersion,
-		int status, String version, ServiceContext serviceContext) {
+		DDMFormInstanceRecordVersion ddmFormInstanceRecordVersion,
+		String formInstanceVersion, int status, User user, String version,
+		ServiceContext serviceContext) {
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java:931]: @rodrigopaulino why did you decide to change the order, putting `user` after `status`, but you kept `serviceContext` after `version`? 
I think we should follow the same pattern as other `LocalServiceImpls` (like [here](https://github.com/liferay/liferay-portal/blob/7eeb47e153c770522d597ed27d0dc975c14e35b1/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMStructureLocalServiceImpl.java#L1561-L1594)) and keep the `User user` at the beginning.

**Comment by carolmariaabb** [comment]
Hey @rodrigopaulino, do you think we need an upgrade process for this?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java (hunk 1 above)
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java:931]: I'm just confused when it comes to these standards. 😑
Thanks for pointing it out, @carolmariaabb.
I'll change it now.

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by rodrigopaulino** [comment]
> Hey @rodrigopaulino, do you think we need an upgrade process for this?

Yes. I decided to provide the solution first, because the customer is waiting for it. I'll open a technical debt to tackle this later, but I do believe this is not a concern for everyone else since this has been like that for a long time.

**Comment by carolmariaabb** [comment]
> > Hey @rodrigopaulino, do you think we need an upgrade process for this?
> 
> Yes. I decided to provide the solution first, because the customer is waiting for it. I'll open a technical debt to tackle this later, but I do believe this is not a concern for everyone else since this has been like that for a long time.

Ok, thanks!

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
Changed it, @carolmariaabb.

**Comment by rodrigopaulino** [comment]
Here it is, @carolmariaabb. https://issues.liferay.com/browse/LPS-159785 😄

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by rodrigopaulino** [comment]
ci:reevaluate:1387401_2759

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**